src/utils/__tests__/googleTagManager.test.tsx

```diff
@@ -3,13 +3,16 @@ import { matchPath } from 'react-router-dom'
 import { renderHook } from '@testing-library/react-hooks'
 
 import { history } from 'src/routes/routes'
-import { getAnonymizedLocation, usePageTracking, trackEvent, GTM_EVENT } from 'src/utils/googleTagManager'
+import { getAnonymizedLocation, usePageTracking, GTM_EVENT } from 'src/utils/googleTagManager'
+import { waitFor } from '@testing-library/react'
 
 jest.mock('react-router-dom', () => ({
   ...jest.requireActual('react-router-dom'),
   matchPath: jest.fn(),
 }))
 
+customElements.define('debug-badge', HTMLElement)
```

Comment:
What is this used for?

Reply:
GTM's `<debug-badge>` web component (injected when previewing GTM) is [used as a flag](https://github.com/gnosis/safe-react/blob/26443542d37c915ce34add9c938f49e3b555857a/src/utils/googleTagManager.ts#L64) to initialise GTM.